recipe/meta.yaml

```diff
@@ -45,32 +45,37 @@ requirements:
     - pyfftw >=0.10.4
     - scipy >=0.14.1
     - trio
-    - opencv
+    - py-opencv
     - scikit-image >=0.12.3
+    - rich
 
 test:
   imports:
-    - fluidimage
-    - fluidimage.calcul
-    - fluidimage.calcul.interpolate
-    - fluidimage.calibration
-    - fluidimage.data_objects
-    - fluidimage.executors
-    - fluidimage.gui
-    - fluidimage.gui.launcher
-    - fluidimage.postproc
-    - fluidimage.preproc
-    - fluidimage.reconstruct
-    - fluidimage.topologies
-    - fluidimage.util
-    - fluidimage.works
+    - cv2  # [not linux]
```

Comment:
Sorry for the less than prompt reply.

My feeling is that this is starting to fail because we moved to qt6 on opencv by "default"

This might be a reason to not do that, but that might take longer to resolve than you would like.

Lets discuss this more on the OpenCV feedstock, but for your feedstock you have two options:

1. Depend during test time on `py-opencv *=headless*`
2. Add the following to your `yum_requirements.txt`:

https://github.com/conda-forge/opencv-feedstock/pull/396/files#diff-e36d294fb902698a9e5c5b991fd921efb1cfb09609fc4db4f94d72759c9e011a

```
mesa-libGL
mesa-libEGL
libglvnd-egl
libglvnd-glx
libglvnd-opengl
```

Reply:
It works fine with `- libopencv *=headless*  # [linux]`.